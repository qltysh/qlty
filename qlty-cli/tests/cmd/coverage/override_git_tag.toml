args = [
  "coverage",
  "publish",
  "--dry-run",
  "--override-commit-sha",
  "2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67",
  "--override-build-id",
  "123",
  "--override-branch",
  "main",
  "--override-commit-time",
  "2025-05-30T05:00:00+00:00",
  "--override-git-tag",
  "v2.0.0-override",
  "lcov.info"
]
bin.name = "qlty"

[env]
remove = ["GITHUB_ACTIONS"]

[env.add]
QLTY_COVERAGE_TOKEN = "123"
GITHUB_REF_TYPE = "tag"
GITHUB_REF_NAME = "v1.0.0"