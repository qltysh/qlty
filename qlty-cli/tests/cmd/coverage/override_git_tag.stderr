qlty [..] [..]-[..] ([..])
[..]
https://qlty.sh/d/coverage

 SETTINGS 

    cwd: [CWD]
    dry-run: true
    override-build-id: 123
    override-commit-sha: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67
    override-commit-time: 2025-05-30T05:00:00+00:00
    override-git-tag: v2.0.0-override

WARNING: Unknown field 'plugins.definitions.dotenv-linter.output_missing' in linters/dotenv-linter/plugin.toml. This will be ignored.
WARNING: Unknown field 'plugins.definitions.ktlint.drivers.lint.cache' in linters/ktlint/plugin.toml. This will be ignored.
WARNING: Unknown field 'plugins.downloads.swiftlint.download_type' in linters/swiftlint/plugin.toml. This will be ignored.
WARNING: Unknown field 'plugins.downloads.swiftlint.github' in linters/swiftlint/plugin.toml. This will be ignored.
WARNING: Unknown field 'plugins.definitions.dotenv-linter.output_missing' in qlty.toml. This will be ignored.
WARNING: Unknown field 'plugins.definitions.ktlint.drivers.lint.cache' in qlty.toml. This will be ignored.
WARNING: Unknown field 'plugins.downloads.swiftlint.download_type' in qlty.toml. This will be ignored.
WARNING: Unknown field 'plugins.downloads.swiftlint.github' in qlty.toml. This will be ignored.
 METADATA 

    CI: unknown
    Reference Type: Tag
    Commit: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67
    Build ID: 123
    Commit Time: 2025-05-30 05:00:00 UTC
    Git Tag: v2.0.0-override

 COVERAGE FILES: 1 

    Coverage File  Format  Size
    lcov.info      lcov    903 B

 COVERAGE DATA 

    1 unique code file path
    1 path is missing on disk (100.0%)

    Missing code files:

      formatter.js

    TIP: Consider using add-prefix or strip-prefix to fix paths
    https://qlty.sh/d/coverage-path-fixing


    Covered Lines:      24
    Uncovered Lines:     2
    Omitted Lines:      26

    Line Coverage:       92.31%

 EXPORTING... 

    Exported: tmp/qlty-coverage/coverage.zip

